### PR TITLE
Add command example to create a capped collection

### PR DESCRIPTION
**Pull request by melkamar**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | 

#### Summary

This PR adds the command to use when using Doctrine with Symfony to create a _capped collection_. 

I spent some time on debugging what exactly needs to be done to have Doctrine listen to the capped collection configuration and it was missing from the documentation.

### CODE REVIEW

File: docs/en/reference/capped-collections.rst [hunk 1]
```diff
@@ -90,3 +90,13 @@ You can drop the collection too if it already exists:
     <?php
 
     $documentManager->getSchemaManager()->dropDocumentCollection('Category');
+
+If using Symfony in addition to Doctrine, use the following command to create the capped collection:
```
Comment (alcaeus) [inline docs/en/reference/capped-collections.rst]: The console commands are available even if not using Symfony, but people need to add them to their console application themselves. I'd reword this:
```suggestion
Use the ``odd:schema:create`` command to create the database schema:
```

File: docs/en/reference/capped-collections.rst [hunk 2]
```diff
@@ -90,3 +90,13 @@ You can drop the collection too if it already exists:
...
+
+.. code-block:: bash
```
Comment (alcaeus) [inline docs/en/reference/capped-collections.rst]: Elsewhere, we use `console` instead of `bash`
```suggestion
.. code-block:: console
```

File: docs/en/reference/capped-collections.rst [hunk 3]
```diff
@@ -90,3 +90,13 @@ You can drop the collection too if it already exists:
...
+
+    php app/console --env=<env> doctrine:mongodb:schema:create
```
Comment (alcaeus) [inline docs/en/reference/capped-collections.rst]: This is how we show console examples elsewhere, as we don't assume people to use the Symfony Bundle:
```suggestion
    $ php mongodb.php odm:schema:create
```

File: docs/en/reference/capped-collections.rst (hunk 1 above)
Comment (Steveb-p) [reply docs/en/reference/capped-collections.rst]: Watch out for typo in ``odd:schema:create`` in suggested changes ;)

Comment (alcaeus) [reply docs/en/reference/capped-collections.rst]: Whoops. Thanks @Steveb-p, I've fixed it.

**Comment by melkamar** [comment]
Applied the suggestions, thanks for them!

**Comment by alcaeus** [comment]
@melkamar can you squash commits? Happy to merge afterwards!

**Comment by alcaeus** [comment]
Thanks @melkamar!
